### ISSUE TITLE
root command in "openedx-dockerfile" patch triggers "permission denied"

### ISSUE THREAD

**Issue by CodeWithEmad** [overhangio/tutor#597]
**Bug description**
Hi, I'm using the tutor for a couple of months and after I upgraded to v13, I noticed that I can't build my own image. neither dev nor production. 
I have a simple Yaml plugin that installs `dnsutils` which worked on the tutor v12.

after a little debugging, I noticed that the {{ patch("openedx-dockerfile") }} is executing by the user called 1000 and looks like it doesn't have the permission to run update/install on the os level. 
 

**How to reproduce**
create a custom YAML plugin like this:
```
name: custom-settings
version: 1.0.0
patches:
  openedx-dockerfile:
    "RUN apt update && apt install -y dnsutils"
```
run `tutor images build openedx`.
in the output:
 ```

 => ERROR [production 28/28] RUN apt update && apt install -y dnsutils                                                                                                                   0.5s
------
 > [production 28/28] RUN apt update && apt install -y dnsutils:
#60 0.316 
#60 0.316 WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
#60 0.316 
#60 0.320 Reading package lists...
#60 0.330 E: List directory /var/lib/apt/lists/partial is missing. - Acquire (13: Permission denied)
------
failed to solve with frontend dockerfile.v0: failed to build LLB: executor failed running [/bin/sh -c apt update && apt install -y dnsutils]: runc did not terminate sucessfully
Error: Command failed with status 1: docker build -t docker.io/overhangio/openedx:13.1.5 /home/ubuntu/maple-base-13-1-5/tutor/env/build/openedx

```
**Environment**
Ubuntu 18.04
Tutor version 13.1.5

**Comment by CodeWithEmad** [comment]
I've fixed it in PR https://github.com/overhangio/tutor/pull/598. please review

**Comment by regisb** [comment]
Closed by #598.
